input-spec.json

```diff
@@ -1420,6 +1420,12 @@
         "type": "float",
         "doc": "Coefficient of friction (global)"
     },
+    {
+        "pointer": "/contact/use_convergent_formulation",
+        "default": false,
```

Comment:
this is not to break backwards compatibility?

Reply:
Yes